src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -755,8 +756,15 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                             .getString(AuthenticationConstants.Browser.RESPONSE_ERROR_MESSAGE);
                     Logger.v(TAG, "Error info:" + errCode + " " + errMessage + " for requestId: "
                             + requestId + correlationInfo);
+                    
+                    ADALError adalError = ADALError.SERVER_INVALID_REQUEST;
```

Comment:
put this error for other resultIntent for BROWSER_CODE_ERROR
And you won't need to check it for null, because it will be required extras as authRequest